tests/ddl_csc2.test/t00_create_table.expected

```diff
@@ -0,0 +1,12 @@
+[CREATE TABLE t1 { schema { cstring c[-1] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { blob c[-1] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { byte c[-1] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { vutf8 c[-1] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { cstring c[-100] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { blob c[-100] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { byte c[-100] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { vutf8 c[-100] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { constants { SIZE = -1 } schema { cstring c[SIZE] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { constants { SIZE = -100 } schema { cstring c[SIZE] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): ERROR AT LINE   0: NEGATIVE ARRAY LENGTH
+[CREATE TABLE t1 { schema { cstring c[1] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): Error at line   0: CSTRINGS ARE \0 TERMINATED SO MUST BE AT LEAST 2 BYTES IN SIZE
+[CREATE TABLE t1 { schema { byte c[0] } }] failed with rc 304 OP #3 BLOCK_SEQ(800): Error at line   0: ZERO LENGTH BYTE ARRAYS ARE NOT ALLOWED
```

Comment:
I removed `OP #3 BLOCK_SEQ(800)` from the error messages. Please rebase and merge it in if it passes all tests.